src/main/resources/tags/sensitive_materials.json

```
@@ -1,11 +1,8 @@
 {
 	"values" : [
 		"#minecraft:saplings",
-		"#minecraft:wooden_pressure_plates",
 		"#slimefun:torches",
-		"minecraft:stone_pressure_plate",
-		"minecraft:light_weighted_pressure_plate",
-		"minecraft:heavy_weighted_pressure_plate",
-		"minecraft:cake"
+		"minecraft:cake",
+		"#slimefun:pressure_plates"
```

Nitpicky, but I'd revert these, so that the order of tags before values is kept.
```suggestion
		"#slimefun:pressure_plates",
		"minecraft:cake"
```